modules/verilator/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://verilator.org/",
+    "maintainers": [
```

Comment:
@QuantamHD @hzeller can I add either one of you as maintainers here? I think it'll streamline any changes needed for [rules_hdl](https://github.com/hdl/bazel_rules_hdl).